### PR TITLE
[platform]: remove sku from build parameter

### PR DESCRIPTION
**Pull request by lguohan**
different sku will be contained in a single one sonic image.
no longer need to specify sku at the build time

### CODE REVIEW

File: files/build_templates/sonic_debian_extension.j2 [hunk 1]
```diff
@@ -118,8 +115,8 @@ sudo cp $IMAGE_CONFIGS/snmp/snmp.yml $FILESYSTEM_ROOT/etc/sonic/
 
 # Generate build version file
 export git_revision=$(git rev-parse --short HEAD)
-export sonic_hwsku={{sonic_hwsku}}
 export debian_version=`cat $FILESYSTEM_ROOT/etc/debian_version`
+export sonic_hwsku=
```
Comment (marian-pritsak) [inline files/build_templates/sonic_debian_extension.j2]: Why leave this line?

Comment (lguohan) [inline files/build_templates/sonic_debian_extension.j2]: will fix

**Comment by stcheng** [comment]
do we want to have minigraph files here? the minigraph files are depending on different kinds of topologies.

**Comment by lguohan** [comment]
that is the sample minigraph, the image can also download one, or the user can update one.

**Comment by lguohan** [comment]
the minigraph can also be generated, platform vendor do not need to check-in.

**Pull request merged**